src/main/resources/db/migration/V1_25__prison_signed_operation_capacity.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE prison_signed_operation_capacity
+(
+    id serial not null constraint signed_operation_capacity_pk primary key,
+    signed_operation_capacity int not null default 0,
+    prison_id varchar(5) not null unique,
```

Comment:
Missing an index on prison_id

Reply:
Unique is adding the index automatically.

Reply:
ok, but prefer to be explicit however fine.